images/win/scripts/Installers/Install-ActionArchiveCache.ps1

```diff
@@ -0,0 +1,20 @@
+################################################################################
+##  File:  Install-ActionArchiveCache.ps1
+##  Desc:  Install Action Archive Cache
+##         from https://github.com/actions/action-versions
+################################################################################
+
+if (-not (Test-Path $env:ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE))
+{
+    Write-Host "Creating action archive cache folder"
+    New-Item -ItemType Directory -Path $env:ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE | Out-Null
+}
+
+Write-Host "Download Latest action-versions archive"
+$latestRelease = Invoke-RestMethod -Uri "https://api.github.com/repos/actions/action-versions/releases/latest"
+[string]$downloadUrl = $latestRelease.assets.browser_download_url -match "action-versions.zip"
+$actionVersionsArchivePath = Start-DownloadWithRetry -Url $downloadUrl -Name "action-versions.zip"
+
+Write-Host "Expand action-versions archive"
+Extract-7Zip -Path $actionVersionsArchivePath -DestinationPath $env:ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE
+Remove-Item $actionVersionsArchivePath -Force
```

Comment:
once Start-DownloadWithRetry is enacted you do not need the line to remove the archive